developer_manual/digging_deeper/profiler.rst

```diff
@@ -0,0 +1,150 @@
+Profiler
+========
+
+Nextcloud offers a built-in profiler that can help you identifying performance issues
+with your Nextcloud application. This feature is enabled starting from Nextcloud 24.
+
+Get the app
+-----------
+
+The profiler application is available on `GitHub <https://github.com/nextcloud/profiler>`_
+you need to clone the stableX branch if you are using Nextcloud X or master if you are
+using the latest development version of Nextcloud.
+
+Enable the app
+--------------
+
+.. warning::
+
+   Do not run this application on production. This will slow down considerably your server
+   as collection profiling information consume a lot of CPU and memory.
+
+To enable the app, you can run the following commands:
+
+.. code-block:: bash
+
+   npm ci && npm run build
+   occ app:enable profiler
+   occ profiler:enable
+
+Disable the app
+---------------
+
+.. code-block:: bash
+
+   occ profiler:disable
+   occ config:system:set debug --value false --type bool # to also disable the debug mode
```

Comment:
Does the app enable it on its own?
There is no set debug true in the enabling command above.